crates/storage/store.rs

```
@@ -470,7 +472,7 @@ impl Store {
                 info!("Received genesis file matching a previously stored one, nothing to do");
                 return Ok(());
             } else {
-                panic!("Tried to run genesis twice with different blocks. Try again after clearing the database. If you're running ethrex as an Ethereum client, run cargo run --release --bin ethrex -- removedb; if you're running ethrex as an L2 run make rm-db-l1 rm-db-l2");
+                panic!("{GENESIS_DIFF_PANIC_MESSAGE}");
```

What's the reason for panicking instead of returning an error?